# Absolute wall time
time wall_time
# Absolute sim time
time sim_time
# Battery charge
float64 current_batt_charge

